src/table/cfg.rs

```diff
@@ -26,54 +26,25 @@ pub struct ConfigTableEntry {
     /// Whether this is a physical or virtual address depends on the table.
     pub address: *const c_void,
 }
-
-/// Entry pointing to the old ACPI 1 RSDP.
-pub const ACPI_GUID: Guid = Guid::from_values(
-    0xeb9d2d30,
-    0x2d88,
-    0x11d3,
-    0x9a16,
-    [0x00, 0x90, 0x27, 0x3f, 0xc1, 0x4d],
-);
+///ryntto oCP0000 1 RSDP.
```

Comment:
Accidental comment change

Reply:
Wild, not sure how those happened but I probably should've caught it, thanks.